packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -95,6 +95,11 @@ _The key elements of the component's public API surface:_
     -   topRight - ![Top Right Quadrant](./Resources/top_right.png)
     -   bottomRight - ![Bottom Right Quadrant](./Resources/bottom_right.png)
     -   `orientation` - represent the orientation of the notch on the wafer map outline (only visual). As only four static orientations are possible, it can be represented by an Enum with the following values: top, bottom, left, right.
+    -   `origin` - represents the origin coordinates of the dies grid for rendering the wafer map. Leaving the value `undefined` will set the value to the minimum corner of the bounding box of the dies coordinates. It will be an object of the type `PointCoordinates` with the following attributes:
+        -   x: number
+        -   y: number
+    -   `numberOfRows` - represents the number of values on the Y axis. Leaving the value `undefined` will set the value to the difference between the maximum and the minimum Y values of the bounding box of the dies coordinates plus one.
+    -   `numberOfColumns` - represents the number of values on the X axis. Leaving the value `undefined` will set the value to the difference between the maximum and the minimum X values of the bounding box of the dies coordinates plus one.
```

Comment:
Can these just be `rows` and `columns`

Reply:
I am worried they can be mistaken for the actual data rows and columns and not just their number

Reply:
Another option. This represents additional metadata of the `dies` property configuration. It describes aspects of the shape of that property.

- `dies-origin-x`, `dies-origin-y`, `dies-rows`, `dies-cols`

The validation property `invalidGridDimensions` introduces the concept of die grid we could use:

- `die-grid-x`, `die-grid-y`, `die-grid-rows`, `die-grid-cols`
- `die-grid-origin-x`, `die-grid-origin-y`, `die-grid-rows`, `die-grid-cols`

I think I lean towards the (`die-grid-x`, `die-grid-y`, `die-grid-rows`, `die-grid-cols`) properties as the name grid lends itself towards the concept of rows and columns and clarifies the layout. Also the shared name prefix emphasizes the relationship between these properties.